tsconfig.json

```diff
@@ -17,7 +17,8 @@
 		"noImplicitThis": false,
 		"esModuleInterop": true,
 		"outDir": "build",
-		"baseUrl": "."
+		"baseUrl": ".",
+		"skipLibCheck": true
```

Comment:
When we ran CI on the file, I got some TS errors due to discrepancies between @types/jest and @types/mocha. Skipping the lib check resolves it.

Reply:
**question:** @elainefigma , please, could you share the TS errors you see? While I do not see any issues with this change, I cannot reproduce the issue and would like to understand why our build is not deterministic.

Please, also, make sure you run `npm ci` instead `npm install` in CI to ensure that the dependencies installation is deterministic and repeatable:
https://betterprogramming.pub/npm-ci-vs-npm-install-which-should-you-use-in-your-node-js-projects-51e07cb71e26

Possibly, the usage of `npm ci` will resolve the issue without changes in `tsconfig.json` 🤞  

Anyway, please, feel free to merge the PR to unblock your work.

Reply:
Good call! Running `npm ci` within the admin/ directory as part of CI resolved this error in our build process. Since we don't need this change, I reverted this bit.